screwdriver.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Job definition block
+# "main" is a default job that all pipelines have
+jobs:
```

Comment:
I'll switch it to using a two-job configuration, so we can show some of the `shared` and `workflow` configuration options